packages/react-flex/package.json

```
@@ -27,29 +27,22 @@
     "@types/react-dom": "16.9.10",
     "@types/webpack-env": "1.16.0",
     "@fluentui/scripts": "^1.0.0",
-    "@fluentui/eslint-plugin": "^1.0.0-beta.2",
+    "@fluentui/eslint-plugin": "^1.0.0",
     "react": "16.8.6",
     "react-app-polyfill": "~1.0.1",
     "react-dom": "16.8.6"
   },
   "dependencies": {
-    "@fluentui/react-compose": "^1.0.0-beta.16",
+    "@fluentui/react-compose": "^1.0.0-alpha.0",
```

```suggestion
    "@fluentui/react-compose": "1.0.0-beta.16",
```

Is this change intentional, `react-compose` has `1.0.0-beta.16` version in the repo.

---

Not intentional, fixed now